dracut/README-testing-changes.rst

```diff
@@ -52,3 +52,6 @@ Using your overlay image
 - In the normal installation environment, you can further check that your changes had the desired
   effect.
 
+| **Note**:
+| If you need to change just a few files, a new-style ``newc:`` syntax allows you don't bother creating an initrd image yourself:
```

Comment:
Noteworthy Fedora has apparently heavily diverged it's Grub flavor from upstream,
as result it doesn't support `newc` syntax for UEFI systems:
https://bugzilla.redhat.com/show_bug.cgi?id=2360458